inference-engine/tests/functional/inference_engine/serialization/single_layer/deformable_convolution.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "shared_test_classes/single_layer/deformable_convolution.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+TEST_P(DeformableConvolutionLayerTest, Serialize) {
+    Serialize();
+}
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
```

Comment:
Based on [spec](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/convolution/DeformableConvolution_1.md) this operation accepts float point element type inputs. Please remove instances of this operation with integer precision